### DISCUSSION TITLE
Windows 8 support

### DISCUSSION THREAD

**Discussion by ZenFlora** [ppy/osu#17875] [Q&A] [answered]
Will windows 8 ever be supported again?

**Comment by smoogipoo** [comment, marked as answer]
Probably not. It's unsupported by Microsoft as of 2016.

**Reply by ZenFlora** [reply]
It pains me since everytime I check if my pc is unsupported I see "Windows 8.1 supported" but not windows 8 :/

**Reply by smoogipoo** [reply]
Is there a reason why you can't upgrade?

**Reply by ZenFlora** [reply]
Lost the registration key, nowhere to be seen.
